awx/ui_next/src/screens/Credential/CredentialList/CredentialListItem.jsx

```diff
@@ -66,16 +72,33 @@ function CredentialListItem({
           id={labelId}
         >
           {canEdit ? (
-            <Tooltip content={i18n._(t`Edit Credential`)} position="top">
-              <Button
-                aria-label={i18n._(t`Edit Credential`)}
-                variant="plain"
-                component={Link}
-                to={`/credentials/${credential.id}/edit`}
-              >
-                <PencilAltIcon />
-              </Button>
-            </Tooltip>
+            <>
+              <Tooltip content={i18n._(t`Edit Credential`)} position="top">
+                <Button
+                  isDisabled={disableButtons}
+                  aria-label={i18n._(t`Edit Credential`)}
+                  variant="plain"
+                  component={Link}
+                  to={`/credentials/${credential.id}/edit`}
+                >
+                  <PencilAltIcon />
+                </Button>
+              </Tooltip>
+              {credential.summary_fields.user_capabilities.copy && (
+                <CopyButton
+                  isDisabled={disableButtons}
+                  css="grid-column: 3"
```

Comment:
Edit the grid definition above instead of passing down this `css` prop. (This prop presently is ignored by the component anyway)